client/src/modules/employees/registration/employees.html

```diff
@@ -0,0 +1,513 @@
+                        is-open="EmployeeCtrl.dateEmbauchePopupIsOpen"
+                        show-button-bar="false"
+                        bh-date-validator="EmployeeCtrl.datepickerOptions"
+                        autocomplete="off">
+                        <span class="input-group-btn">
+                          <button type="button" class="btn btn-default" ng-click="EmployeeCtrl.dateEmbauchePopupIsOpen = !EmployeeCtrl.dateEmbauchePopupIsOpen">
+                            <i class="fa fa-calendar-plus-o"></i>
+                          </button>
+                        </span>
+                      </div>
+
+                      <div class="help-block" ng-messages="DetailForm.dateEmbauche.$error" data-date-error ng-if="EmployeeRegistrationForm.$submitted && DetailForm.dob.$invalid">
+                        <div ng-messages-include="modules/templates/dates.messages.tmpl.html"></div>
+                      </div>
+                    </div>
+                  </div>
+                  <!-- Employee number of spouse-->
+                  <div class="form-group"
+                    ng-class="{'has-error' : OptionalForm.nb_spouse.$invalid && EmployeeRegistrationForm.$submitted}">
+
+                    <label class="col-md-3 control-label">{{ "FORM.LABELS.NB_SPOUSE" | translate }}</label>
+                    <div class="col-md-9">
+                      <input
+                        type="number"
+                        class="form-control"
+                        name="nb_spouse"
+                        ng-model="EmployeeCtrl.employee.nb_spouse"
+                        autocomplete="off">
+                     <div class="help-block" ng-messages="OptionalForm.nb_spouse.$error" ng-show="EmployeeRegistrationForm.$submitted">
+                      <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
+                     </div>
+                    </div>
+                  </div>
+
+                  <!-- Employee number of child-->
+                  <div class="form-group"
+                    ng-class="{'has-error' : OptionalForm.nb_enfant.$invalid && EmployeeRegistrationForm.$submitted}">
+
+                    <label class="col-md-3 control-label">{{ "FORM.LABELS.NB_CHILD" | translate }}</label>
+                    <div class="col-md-9">
+                      <input
+                        type="number"
+                        class="form-control"
+                        name="nb_enfant"
+                        ng-model="EmployeeCtrl.employee.nb_enfant"
+                        autocomplete="off">
+                     <div class="help-block" ng-messages="OptionalForm.nb_enfant.$error" ng-show="EmployeeRegistrationForm.$submitted">
+                      <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
+                     </div>
+                    </div>
+                  </div>
+
+                  <!-- Employee service-->
+                  <div class="form-group"
```

Comment:
Okay, that makes sense.

We may need to separate the idea of services at Patient Invoice from these services.  We wouldn't want Patient Invoices to go to the SERVICES GENERAUX .. that data would be messy to clean up.